<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XMall</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background:rgb(40, 44, 50);
color:white;
font-family:Microsoft YaHei;
text-align: center;
}

QFrame{
border:sold 10px rgba(0 ,0, 0)
}


QLineEdit{
background-color:rgb(58. 104. 148);
}

QPushButton{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-color:white;
border-radius:3px;
border:0px solid ;
}

QPushButton:pressed{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-radius:3px;
border:1px solid;
}



QLineEdit{
border: 1px solid #A0A0A0;
background-color:rgb(65, 65, 65);
color:white;
}


QLineEdit:hover { 
    border: 1px solid #298DFF;
    border-radius: 3px;
    selection-color: #F2F2F2;
}

QLineEdit[EchoMode=&quot;2&quot;] {lineedit-password-character:42;}

QMenu::item 
{
     background-color:rgb(42, 42, 42);
}
 
/*选择项设置*/
QMenu::item:selected 
{
     background-color: rgb(36, 58, 99);
}
QMenuBar::item:selected{
background-color:rgb(35, 60, 91);
color:white;
}
QMenuBar::item{
color:rgb(134, 134, 134);
}
QTabWidget::pane{
border:none;
background: transparent;

}
QTabWidget::tab-bar {
border: none;   
}
QTabBar::tab {
border: none;
color: rgb(158, 158, 158);
font:22px;
background:transparent;
height: 28px;
min-width: 85px;
margin-right: 5px;
padding-left: 5px;
padding-right: 5px;

}
QTabBar::tab:hover {
background: transparent;
color:rgb(225, 225, 225);

}
QTabBar::tab:selected {
color: rgb(255, 255, 255);
background: transparent;
}

QLabel{
background-color:rgb(58, 82, 127);
}
#profileTabWidget::pane{
border:2px solid rgb(59, 98, 134);
border-radius:10px;
}

QStatusBar QLabel{
border:none;
background:transparent;
}

QCheckBox::indicator, QRadioButton::indicator, QTreeView::indicator {
width: 16px;
height: 16px;
background-color: #111111;
border: 1px solid #333333;
}

QRadioButton::indicator {
border-radius: 8px;
}

QCheckBox::indicator::checked, QRadioButton::indicator::checked, QTreeView::indicator::checked {
background-color: qradialgradient(cx:0.5, cy:0.5, fx:0.25, fy:0.15, radius:0.3, stop:0 #BBBBBB, stop:1 #111111);
}

QCheckBox::indicator:disabled, QRadioButton::indicator:disabled, QTreeView::indicator:disabled {
background-color: #444444;
}

QCheckBox::indicator::checked:disabled, QRadioButton::indicator::checked:disabled, QTreeView::indicator::checked:disabled {
background-color: rgb(0, 99, 148)
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="centralTabWidget">
      <property name="styleSheet">
       <string notr="true">
QLineEdit{
background-color:rgb(58. 104. 148);
}

QPushButton{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-color:white;
border-radius:3px;
border:0px solid ;
}

QPushButton:pressed{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-radius:3px;
border:1px solid;
}</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="categoryTab">
       <property name="styleSheet">
        <string notr="true">*{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 44, 50), stop:1 rgb(49, 97, 117))

}
QMenu::item 
{
     background-color:rgb(42, 42, 42);
}
 
/*选择项设置*/
QMenu::item:selected 
{
     background-color: rgb(36, 58, 99);
}
QMenuBar::item:selected{
background-color:rgb(51, 124, 188);
color:white;
}
QMenuBar::item{
color:white;
font:20px;
}
QLabel{
background:transparent
}</string>
       </property>
       <attribute name="title">
        <string>类目管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10" stretch="2,11,2,11">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>一级类目</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
          <item>
           <widget class="CategoryRefreshWidget" name="categoryWidget_1" native="true"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,2,1">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>新增一级类目</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="categoryLineEdit_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>35</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,2,3">
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="categoryPushButton_1">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">

QPushButton{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-color:white;
border-radius:3px;
border:0px solid ;
}

QPushButton:pressed{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-radius:3px;
border:1px solid;
}</string>
                </property>
                <property name="text">
                 <string>保存</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>二级类目</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
          <item>
           <widget class="CategoryRefreshWidget" name="categoryWidget_2" native="true"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,1,1,2,2">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>新增二级类目</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>选择父目录</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="categoryComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="categoryLineEdit_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>35</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,2,3">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="categoryPushButton_2">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">
QPushButton{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-color:white;
border-radius:3px;
border:0px solid ;
}

QPushButton:pressed{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-radius:3px;
border:1px solid;
}</string>
                </property>
                <property name="text">
                 <string>保存</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
       <action name="settingAction">
        <property name="text">
         <string>设置</string>
        </property>
       </action>
       <action name="switchAccountAction">
        <property name="text">
         <string>切换账户</string>
        </property>
       </action>
       <action name="exitAction">
        <property name="text">
         <string>退出</string>
        </property>
       </action>
       <action name="serviceAction">
        <property name="text">
         <string>XMall客服</string>
        </property>
       </action>
      </widget>
      <widget class="QWidget" name="productTab">
       <property name="styleSheet">
        <string notr="true">*{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 44, 50), stop:1 rgb(49, 97, 117))

}</string>
       </property>
       <attribute name="title">
        <string>商品管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,14,1,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="9,1">
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>刷新</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>管理现存商品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ProductRefreshWidget" name="productRefreshWidget" native="true"/>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>添加商品</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,2,7">
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="addProductButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">
QPushButton{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-color:white;
border-radius:3px;
border:0px solid ;
}

QPushButton:pressed{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-radius:3px;
border:1px solid;
}</string>
              </property>
              <property name="text">
               <string>添加商品</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="orderTab">
       <property name="styleSheet">
        <string notr="true">*{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 44, 50), stop:1 rgb(49, 97, 117))

}

QPushButton{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-color:white;
border-radius:3px;
border:0px solid ;
}

QPushButton:pressed{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-radius:3px;
border:1px solid;
}
</string>
       </property>
       <attribute name="title">
        <string>订单管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="14,1">
        <item>
         <widget class="OrderViewWidget" name="orderView" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <property name="spacing">
           <number>40</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="waitPayPushButton">
            <property name="text">
             <string>待付款</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="waitDeliveryPushButton">
            <property name="text">
             <string>待发货</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="waitRecvPushButton">
            <property name="text">
             <string>待收货</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="waitRefundPushButton">
            <property name="text">
             <string>退款申请中</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="allPushButton">
            <property name="text">
             <string>全部</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="freightTab">
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 44, 50), stop:1 rgb(49, 97, 117))

}

QMenu::item 
{
     background-color:rgb(42, 42, 42);
}
 
/*选择项设置*/
QMenu::item:selected 
{
     background-color: rgb(36, 58, 99);
}
QMenuBar::item:selected{
background-color:rgb(51, 124, 188);
color:white;
}
QMenuBar::item{
color:white;
font:20px;
}
QLabel{
background:transparent
}</string>
       </property>
       <attribute name="title">
        <string>运费模板</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8" stretch="2,1">
          <item>
           <widget class="FreightRefreshWidget" name="freightRefreshWidget" native="true"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,1,1,1,1">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>新增模板</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="freightDoubleSpinBox">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>45</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Microsoft YaHei</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="suffix">
               <string>￥</string>
              </property>
              <property name="maximum">
               <double>10000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="5,3,5">
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="freightPushButton">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">

QPushButton{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-color:white;
border-radius:3px;
border:0px solid ;
}

QPushButton:pressed{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-radius:3px;
border:1px solid;
}</string>
                </property>
                <property name="text">
                 <string>添加</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuXMall">
    <property name="title">
     <string>XMall</string>
    </property>
    <addaction name="switchAccountAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menuXMall"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="settingAction">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="switchAccountAction">
   <property name="text">
    <string>切换账户</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="serviceAction">
   <property name="text">
    <string>XMall客服</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProductRefreshWidget</class>
   <extends>QWidget</extends>
   <header>productrefreshwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrderViewWidget</class>
   <extends>QWidget</extends>
   <header>orderviewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CategoryRefreshWidget</class>
   <extends>QWidget</extends>
   <header>categoryrefreshwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FreightRefreshWidget</class>
   <extends>QWidget</extends>
   <header>freightrefreshwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
